t/CorrectPod.pm

```diff
@@ -0,0 +1,36 @@
+package t::CorrectPod;
```

Comment:
All of these test helper files would be best inside their own subdirectory

Reply:
In the beginning, they were on separate subdirectories but I checked some similar public packages on the `cpan` and I found out they keep these kinds of helper files into the same directory as test files. I personally agree with you on this but I've tried to follow the confirmed practices.